widgets/common/src/inputs/fields/BooleanInputField.tsx

```diff
@@ -0,0 +1,25 @@
+import RevertToDefaultIcon from './RevertToDefaultIcon';
+import { RevertableInputFieldProps } from './types';
+
+export default function BooleanInputField(props: RevertableInputFieldProps) {
+    const { Form } = Stage.Basic;
+    const { name, value, onChange } = props;
```

Comment:
We could destructurize props as it has been done in other, extracted input fields 😉 

```suggestion
export default function BooleanInputField({
    name,
    value,
    onChange
}: RevertableInputFieldProps) {
    const { Form } = Stage.Basic;
```

Reply:
See L22 - `props` object is used there directly.

Reply:
Oh yea, you're right!
Thanks for the clarification :)

Reply:
Oh yea, you're right!
Thanks for the clarification :)